docs/glossary.md

```diff
@@ -5,53 +5,38 @@ sidebar_label: Glossary
-| Component | Entity that is the subject of a Control, eg a Virtual Machine instance. A Component's status is made up of collective results of ComponentRequirements pertaining to that Component. |
-| ComponentAttestation | The Attestations carried out on a Component. |
-| ComponentAssessment | An instance of an Implementation for a Component. |
-| ComponentControl | An instance of a Control linked to a Component. A ComponentControl asks a ComponentAssessment what the results of its ComponentAttestations were. |
-| Detective Control | A control that records that a qualifying event has taken place, usually with a view to following up with a check that the event was valid. |
-| DORA | Digital Operational Resilience Act Digital Operational Resilience Act (DORA) - Regulation (EU) 2022/2554|
-| FINOS | Fintech Open Source Foundation |
-| GRC | Governance, Risk and Control |
-| Assessment | A specific Implementation of a Control, eg a cronjob that kills any processes that listen on port 22 every minute might implement a Reactive Control.|
-| KRI | Key Risk Indicator: a metric for measuring the probability of an event and its consequences will exceed the organisation's risk appetite |
-| OSCAL | Open Security Controls Assessment Language OSCAL - Open Security Controls Assessment Language |
-| Preventative Control | A Control that prevents an event from taking place, eg an organisational policy that prevents an S3 bucket being exposed to the internet. |
-| Reactive Control | A Control that takes corrective action when an event takes place, eg shutting down a VM that has attached an unencrypted disk. |
-| Requirement | See Control |
-| RPO | Recovery Point Objective |
-| SMF24 | |
-| Senior Management Function 24 | the person in an organisation held responsible for operational failings. |
-| Variance | |
-
+| Name                  | Definition                                                                                                                                                                                                                               |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Assessment            | The overall process of evaluating a specific system, control, or process. It includes planning, executing various tests or checks, and drawing conclusions about compliance, performance, or quality.                                    |
+| Attestation           | A formal declaration or statement that confirms that something is true, based on the Assessment and its Observations.                                                                                                                    |
```

Comment:
`Attestation           | A formal declaration or statement confirming the truth of something, based on the assessment and its observations.`